docs/execution-providers/MIGraphX-ExecutionProvider.md

```diff
@@ -1,54 +1,88 @@
 ---
 title: AMD - MIGraphX
 description: Instructions to execute ONNX Runtime with the AMD MIGraphX execution provider
-grand_parent: Execution Providers
-parent: Community-maintained
-nav_order: 4
+parent: Execution Providers
+nav_order: 10
 redirect_from: /docs/reference/execution-providers/MIGraphX-ExecutionProvider
 ---
 
 # MIGraphX Execution Provider
 {: .no_toc }
 
-The [MIGraphX](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/) execution provider uses AMD's Deep Learning graph optimization engine to accelerate ONNX model on AMD GPUs. 
+The [MIGraphX](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/) execution provider uses AMD's Deep Learning graph optimization engine to accelerate ONNX model on AMD GPUs.
 
 ## Contents
 {: .no_toc }
 
-* TOC placeholder
```

Comment:
Can you add lines 18+19 back to display the table of contents for the page?

Reply:
Sure thing.

Reply:
Done